src/lsp-server/CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to the Materialize Language Server Protocol (LSP)
+
+## Developing the Extension
+
+Thank you for your interest in the Materialize LSP! Contributions of many kinds are encouraged and most welcome.
+
+If you have questions, please create a GitHub issue.
+
+### Requirements
+* Rust
+
+### Building the Extension
+
+1. Clone the repository:
+```bash
+git clone https://github.com/MaterializeInc/materialize.git
+```
+1. Test the LSP:
```

Comment:
Nit: this shows how to test the extension, not how to build it (as the header indicates).